### PR TITLE
Feature/package renaming

### LINKED ISSUE #118: rename package to include xpanse
1. Update package name.
2. Update module names  -yes. Everything except OCL and OCL Loader
3. Update container name? - yes
4. update API URI? - yes
5. Documents.
6. At the end also update the github project name.

### PR DESCRIPTION
**Pull request by iskey**
fix #118 
- [x] 1. Rename names of docker images from osc to xpanse.
- [x] 2. Update readme files (including Open Services Cloud).
- [x] 3. Update banner.txt
- [x] 4. ~~Update demo GUI (Need more discussion, not in this US).~~
- [x] 5.  ~~Update GitHub pages (Need more discussion, not in this US).~~

### CODE REVIEW

File: website/docs/architecture.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ sidebar_position: 2
...
-Open Services Cloud is a framework reshaping the cloud service ecosystem:
+xpanse is a framework reshaping the cloud service ecosystem:
 
 * the cloud users can find the same services across different cloud providers, exactly the same services provided by
-  Open Services Cloud.
+  xpanse.
 * the software vendors can easily create native and portable managed services for their software, including seamless and
   low level integration with cloud provider services. In addition of software artifacts, software vendor describe the
-  service using the Open Services Cloud Configuration Language (OCL).
+  service using the xpanse Configuration Language (OCL).
```
Comment (swaroopar) [inline website/docs/architecture.md:13]: OCL must be still used as it is.

**Review by swaroopar — CHANGES_REQUESTED** [review]
1. We can also rename "demo" folder to "mock-ui". It was suggested in the Eclipse foundation meeting.

File: runtime/src/main/resources/banner.txt [hunk 2]
```diff
@@ -1,8 +1,7 @@
-  ___                     ____                  _                  ____ _                 _
- / _ \ _ __   ___ _ __   / ___|  ___ _ ____   _(_) ___ ___  ___   / ___| | ___  _   _  __| |
-| | | | '_ \ / _ \ '_ \  \___ \ / _ \ '__\ \ / / |/ __/ _ \/ __| | |   | |/ _ \| | | |/ _` |
-| |_| | |_) |  __/ | | |  ___) |  __/ |   \ V /| | (_|  __/\__ \ | |___| | (_) | |_| | (_| |
- \___/| .__/ \___|_| |_| |____/ \___|_|    \_/ |_|\___\___||___/  \____|_|\___/ \__,_|\__,_|
-      |_|
+   _  __    ____   ____ _   ____    _____  ___
```
Comment (swaroopar) [inline runtime/src/main/resources/banner.txt:1]: is there a space here between "x" and "p"?

File: website/docs/architecture.md (hunk 1 above)
Comment (iskey) [reply website/docs/architecture.md:13]: Yes, we have discussed this, but I forgot to push it up.
BYW, Should we only rename the "demo" folder to "mock-ui", but leave the contents as it is?

File: runtime/src/main/resources/banner.txt (hunk 2 above)
Comment (iskey) [reply runtime/src/main/resources/banner.txt]: Not a space there, actually It was generated automatically by tools. 
I can fix it manually to make it seems better.

File: website/docs/architecture.md (hunk 1 above)
Comment (swaroopar) [reply website/docs/architecture.md:13]: yes, only the folder must be renamed.

**Review by swaroopar — APPROVED** [review]
Looks good.

**Pull request merged**